scripts/bert/finetune_classifier.py

```diff
@@ -105,52 +105,57 @@
     type=float,
     default=0.1,
     help='ratio of warmup steps used in NOAM\'s stepsize schedule')
-parser.add_argument(
-    '--log_interval',
-    type=int,
-    default=10,
-    help='report interval')
-parser.add_argument(
-    '--max_len',
-    type=int,
-    default=128,
-    help='Maximum length of the sentence pairs')
+parser.add_argument('--log_interval',
+                    type=int,
+                    default=10,
+                    help='report interval')
+parser.add_argument('--max_len',
+                    type=int,
+                    default=128,
+                    help='Maximum length of the sentence pairs')
 parser.add_argument(
     '--seed', type=int, default=2, help='Random seed')
 parser.add_argument(
     '--accumulate',
     type=int,
     default=None,
-    help='The number of batches for gradients accumulation to simulate large batch size. '
-         'Default is None')
-parser.add_argument(
-    '--gpu', type=int, default=None, help='Which gpu for finetuning.')
+    help=
+    'The number of batches for gradients accumulation to simulate large batch size. '
+    'Default is None')
+parser.add_argument('--gpu',
+                    type=int,
+                    default=None,
+                    help='Which gpu for finetuning.')
 parser.add_argument(
     '--task_name',
     type=str,
     choices=tasks.keys(),
     help='The name of the task to fine-tune. Choices include MRPC, QQP, '
-         'QNLI, RTE, STS-B, CoLA, MNLI, WNLI, SST.')
-parser.add_argument(
-    '--bert_model',
-    type=str,
-    default='bert_12_768_12',
-    choices=['bert_12_768_12', 'bert_24_1024_16', 'roberta_12_768_12', 'roberta_24_1024_16'],
-    help='The name of pre-trained BERT model to fine-tune')
-parser.add_argument(
-    '--bert_dataset',
-    type=str,
-    default='book_corpus_wiki_en_uncased',
-    choices=['book_corpus_wiki_en_uncased', 'book_corpus_wiki_en_cased',
-             'openwebtext_book_corpus_wiki_en_uncased', 'wiki_multilingual_uncased',
-             'wiki_multilingual_cased', 'wiki_cn_cased',
-             'openwebtext_ccnews_stories_books_cased'],
-    help='The dataset BERT pre-trained with.')
-parser.add_argument(
-    '--pretrained_bert_parameters',
-    type=str,
-    default=None,
-    help='Pre-trained bert model parameter file.')
+    'QNLI, RTE, STS-B, CoLA, MNLI, WNLI, SST.')
+parser.add_argument('--bert_model',
+                    type=str,
+                    default='bert_12_768_12',
+                    choices=[
+                        'bert_12_768_12', 'bert_24_1024_16',
+                        'roberta_12_768_12', 'roberta_24_1024_16'
+                    ],
+                    help='The name of pre-trained BERT model to fine-tune')
+parser.add_argument('--bert_dataset',
+                    type=str,
+                    default='book_corpus_wiki_en_uncased',
+                    choices=[
```

Comment:
I think we need to add an API `nlp.data.list_datasets()` to list all available datasets in gluonnlp. Otherwise every time a new model is added, we need to revise the choice list in the script..